xunit-autolabeler-v2/ast_parser/core/cli_yaml_errors.py

```diff
@@ -0,0 +1,108 @@
+# Copyright 2020 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This file contains string descriptions of YAML validation errors. These
+errors are represented as classes to a) make message modification easier
+and b) enable direct isinstance() checks.
+"""
+
+from dataclasses import dataclass
+
+
+@dataclass
```

Comment:
Don't init with the default repr if you are going to overwrite it.
@dataclasses.dataclass(repr=False)

Reply:
Done.